hydroflow/examples/two_pc/coordinator.rs

```diff
@@ -118,38 +134,72 @@ pub(crate) async fn run_coordinator(opts: Opts, subordinates: Vec<String>) {
             .push_to(p1_send_push),
     );
 
+    // Count the size of subordinates.
+    hf.add_subgraph_stratified(
+        "count the size of subordinates",
+        0,
+        subordinates_out_tee_4_pull
+            .flatten()
+            .map_scan(0u32, |counter, _| {
+                *counter += 1;
+                *counter
+            })
+            .pull_to_push()
+            .map(Some)
+            .push_to(subordinate_count_1_push),
+    );
+
+    // Take the max of subordinate counts.
```

Comment:
Can we add a comment to explain the reasoning for MAX(COUNT) here?

Reply:
I believe the issue is that we generate a running count, and need to choose the highest number we ever issue. Yes? This won't be clear without a comment :-)

Reply:
Also note in the code that this is inefficient unless we do a lot of batching.